### DISCUSSION TITLE
What are the best practices for seeders after migration?

### DISCUSSION THREAD

**Discussion by yusuke2930** [fastapi/full-stack-fastapi-template#1810] [Questions] [answered]
I want to insert master data which is seed data after migration.
Where would you like to implement this?
`init_db.py`?

**Comment by YuriiMotov** [comment, marked as answer]
`init_db` function from `backend/app/core/db.py` serves this specific purpose.
In default configuration. it's run by `prestart` service right after applying migrations (see `backend/scripts/prestart.sh`)
